modAionImpl/src/org/aion/zero/impl/sync/TaskImportBlocks.java

```diff
@@ -306,22 +315,43 @@ private ImportResult importBlock(Block b, String displayId, SyncMode mode) {
                     b.getTransactionsList().size(),
                     b.getTimestamp(),
                     importResult,
-                    t2 - t1,
+                    (t2 - t1) / DIVISOR_MS,
                     chain.getTotalDifficulty());
         } else {
             // not printing this message when the state is in fast mode with no parent result
             // a different message will be printed to indicate the storage of blocks
-            if (log.isInfoEnabled() && (importResult != ImportResult.NO_PARENT)) {
+            if (importResult != ImportResult.NO_PARENT) {
                 log.info(
                         "<import-status: node = {}, hash = {}, number = {}, txs = {}, result = {}, time elapsed = {} ms>",
                         displayId,
                         b.getShortHash(),
                         b.getNumber(),
                         b.getTransactionsList().size(),
                         importResult,
-                        t2 - t1);
+                        (t2 - t1) / DIVISOR_MS);
             }
         }
+
+        if (surveyLog.isInfoEnabled()) {
+            if (importResult.isBest()) {
+                long import_time = (t2 - t1);
```

Comment:
minor: I would prefer to see this converted into an object for tracking the different values, where each time update triggers the necessary computations